docs/source/endpoints/navroot.md

```diff
@@ -0,0 +1,159 @@
+---
+html_meta:
+  "description": "Navigation root is a concept that provides a way to root catalog queries, searches, and breadcrumbs in Plone."
+  "property=og:description": "Navigation root is a concept that provides a way to root catalog queries, searches, and breadcrumbs in Plone."
+  "property=og:title": "Navigation Root"
+  "keywords": "Plone, plone.restapi, REST, API, site, navigation root"
+---
+
+(navigation-root-label)=
+
+# Navigation root
+
+Plone has a concept called {term}`navigation root` which provides a way to root catalog queries, searches, breadcrumbs, and so on in a given section of the site.
+This feature is useful when working with subsites or multilingual sites, because it allows the site manager to restrict searches or navigation queries to a specific location in the site.
+
+This navigation root information is different depending on the context of the request.
+For instance, in a default multilingual site when browsing the contents inside a language folder (`www.domain.com/en`), the context is `en` and its navigation root will be `/en/`.
+In a non-multilingual site, the context is the root of the site (`www.domain.com`) and the navigation root will be `/`.
+
+To get the information about the navigation root, the REST API has a `@navroot` contextual endpoint, which will return the correct information about it.
+For instance, send a `GET` request to the `@navroot` endpoint in the root of the site:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_get.req
+```
+
+The response will contain the navigation root information with the site:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_get.resp
+:language: http
+```
+
+In a multilingual site (where the language root folders are navigation root objects), the endpoint will work in a similar way.
+
+If you request the `@navroot` of a given content type in the site:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_content_get.req
+```
+
+The response will contain the navigation root information with the site:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/navroot_standard_site_content_get.resp
+:language: http
+```
```

Comment:
Check the preceding request and response in the preview (https://deploy-preview-1465--plone-restapi.netlify.app/endpoints/navroot.html). I don't see a language root such as `/en/` for these examples, even though the narrative text claims it is multilingual.

Reply:
Just in this paragraph I didn't want to talk about multilingual, but the sentence of then 34th line creates confusion. I just want to say that you can request the @navroot of any content on the site, and you will get the response. 

I will delete the 34th line sentence (`In a multilingual site where the language root folders are navigation root objects, the endpoint will work in a similar way.`) because it is somehow a duplicate of a later line (`In a multilingual site, the root of the site will work as usual`), and I will merge both of them.